src/app/home/components/home.component.ts

```diff
@@ -17,4 +17,8 @@ export class HomeComponent {
       width: constants.dialogWidth
     });
   }
+
+  playAgainstTheBot(): void {
```

Comment:
It's a great idea, but I would prefer to create a ticket and remove this again. I am already sharing with friends, and I would like to avoid the confusion of a non-functional button :)

Reply:
I see, removed